webapp-django/crashstats/manage/admin.py

```diff
@@ -218,9 +220,80 @@ def debug_view(request):
 
 @superuser_required
 def products(request):
+    """Lists products from the products table"""
     context = {}
     api = ProductsMiddleware()
     context['products'] = api.get()['hits']
     context['title'] = 'Products'
 
     return render(request, 'admin/products.html', context)
+
+
+@superuser_required
+def buildhub_check(request):
```

Comment:
This is "quick and dirty" and copies some code from the BetaVersionRule. It's temporary and will get removed when we remove the `product_versions` table and all its friends. At that point, we can replace it with a page that checks recent crash reports with "b0" to determine why they're stinky. (ha! get it--b0! stinky!)